src/main/kotlin/io/UstMode1.kt

```diff
@@ -0,0 +1,111 @@
+package io
+
+import external.JsZip
+import external.JsZipOption
+import kotlinx.coroutines.await
+import model.DEFAULT_METER_HIGH
+import model.DEFAULT_METER_LOW
+import model.ExportNotification
+import model.ExportResult
+import model.Feature
+import model.Format
+import model.Project
+import model.Track
+import org.khronos.webgl.Uint8Array
+import org.w3c.files.Blob
+import process.pitch.UtauMode1NotePitchData
+import process.pitch.UtauMode1TrackPitchData
+import process.pitch.pitchToUtauMode1Track
+import util.encode
+import util.getSafeFileName
+import util.padStartZero
+import util.toFixed
+
+//Only contains generator, as Mode2 generator has not been implemented.
+object UstMode1 {
```

Comment:
this is, emm, as you say we should add UI for selection in Configurator but not adding a new format.
Also as UTAU can convert mode2 data to mode1, it's not so urgently required, so we can leave it later (together with the UI change).

Reply:
I see...Well, should we just remove the UST_MODE1 section in the Format.kt, and keep these code for feature merge? Or provide the seperate format generator here as an option to obtain other software's pitch currently?

Reply:
We are not going to need this format so please remove it. For the code about generation, we can check commit history to get it back when we start to do this part later